_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,205 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements that are [exposed to assistive technologies](#exposed-to-assistive-technologies) and have a [non-empty](#non-empty) accessible name.
+
+**Note:** 
+- Nested `iframe` contents treat the parent `iframe` element as the [document context](#document-context).
+
+### Expectation
+
+Each target element has an [accessible name](#accessible-name) that is unique within the [document context](#document-context).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [http://www.w3.org/TR/WCAG20-TECHS/H64.html](http://www.w3.org/TR/WCAG20-TECHS/H64.html)
```

Comment:
Clean up links to have nice link texts instead of just URLs (e.g. the title/H1 of the destination document)